### ISSUE TITLE
"Clear" Taskbar not working in Windows 11

### ISSUE THREAD

**Issue by ghost** [TranslucentTB/TranslucentTB#527]
I just upgraded my OS from Windows 10 to Windows 11 and now my Translucent TB isn't working at all. Everytime I try to set my Taskbar to "Clear" all I get is a solid gray. I have both "Desktop" and "Task View Open" set to "Clear" and nothing. Tried uninstalling, re-installing, still nothing. Anyone know the solution here?

**Comment by TJ-programmer** [comment]
did you find solution bro??

**Comment by mcdonamw** [comment]
Have you actually enabled transparency in the Personalization > Colors settings in Windows 11?  It's off by default.  This must be enabled in order for it to work.

**Comment by AgrockP** [comment]
> Have you actually enabled transparency in the Personalization > Colors settings in Windows 11? It's off by default. This must be enabled in order for it to work.

Hey, I'm also facing the same issue, and yeah I've enabled transparency effects from the colors settings in Windows 11

**Comment by AgrockP** [comment]
> > Have you actually enabled transparency in the Personalization > Colors settings in Windows 11? It's off by default. This must be enabled in order for it to work.
> 
> Hey, I'm also facing the same issue, and yeah I've enabled transparency effects from the colors settings in Windows 11

So, i saw that if you're on insider builds, these won't work properly, and i am on insider builds

**Comment by TJD269** [comment]
> > > Have you actually enabled transparency in the Personalization > Colors settings in Windows 11? It's off by default. This must be enabled in order for it to work.
> > 
> > 
> > Hey, I'm also facing the same issue, and yeah I've enabled transparency effects from the colors settings in Windows 11
> 
> So, i saw that if you're on insider builds, these won't work properly, and i am on insider builds

Thanks for posting this, I am on an insider build and can confirm this is happening on my work PC with Windows 11 insider. I am hoping this doesn't happen in stable since I just bought an OLED and will be switching my home PC to Windows 11 for all the HDR goodies.

**Comment by sylveon** [comment]
For insider build, see #537

**Comment by oleksis** [comment]
> For insider build, see #537

I'm using TTB from [source](82ba27d7bffb2fd5312a013585360a767745a03a) with **Windows Dev 22579** and the **Insider SDK 22579** compiled with **Visual Studio 2022**. 

Applied the `vivetool` like say [here](https://github.com/TranslucentTB/TranslucentTB/issues/537#issuecomment-1053807594) and "Clear" still not working

Any advice?

**Comment by TJD269** [comment]
I couldn't get it to work either and my Windows Update has been borked too,
so I can't change versions. Tried the same steps as you.

**Comment by gisellecdantas** [comment]
the same here. I tried everything, but my tb is still opaque.

**Comment by sylveon** [comment]
There hasn't been a fix commited yet to the repo.

**Comment by gisellecdantas** [comment]
i checked the updates on microsoft updates and installed them, and now everything's working..

**Comment by oleksis** [comment]
You update Windows ? What version you are using?

Windows:

TTB:

**Comment by gisellecdantas** [comment]
![versão windows](https://user-images.githubusercontent.com/97754120/159839839-1af9bec6-c74f-41c0-be6d-f4ced9f8203f.jpg)

**Comment by MarcusCCesar** [comment]
This build does not activate the (Clear) features of TranslucentTB...
Edição	Windows 11 Home Single Language
Versão	22H2
Instalado em	‎24/‎03/‎2022
Compilação do SO	22581.1
Experiência	Windows Feature Experience Pack 1000.22581.1.0

**Comment by Crilor** [comment]
I confirm Translucency lost after 22581.1 insider update

**Comment by xGiorgo** [comment]
I am also on the insider preview on windows 11 and it doesn´t work properly.
I selected ´´clear´´ but it´s grey

**Comment by Soulzityr** [comment]
Hello,

I am also on Version 22H2, Build 22581.1 and TranslucentTB does not set my taskbar to "Clear". In fact, none of the settings work. I do believe that the issue linked [here](https://github.com/TranslucentTB/TranslucentTB/issues/537) is responsible.

**Comment by lone-assassin-aj** [comment]
Hey, to the people still looking for the fix, there is a workaround I found which seems to work fine on my end. Hope this helps:

1. Download RoundedTB from MS Store and open the app.
2. Margin - 3, Corner Radius - 0 || Advanced > Check the box for TranslucentTB compatibility.
3. Apply.

**Comment by SkullCandyTV** [comment]
> Hey, to the people still looking for the fix, there is a workaround I found which seems to work fine on my end. Hope this helps:
> 
> 1. Download RoundedTB from MS Store and open the app.
> 2. Margin - 3, Corner Radius - 0 || Advanced > Check the box for TranslucentTB compatibility.
> 3. Apply.

This doesn't work

**Comment by turgayakdag** [comment]
again not working tool on Windows 11 Insider Preview 10.0.22623.730 (ni_release)

**Comment by Willxxzinho** [comment]
again not working
![image](https://user-images.githubusercontent.com/117618835/200224225-4878cb18-d012-444c-bc2b-6d8ac5ad522c.png)

**Comment by sylveon** [comment]
#537

**Comment by abhinav9799** [comment]
> did you find solution bro??

simply go to settings>windows updates>update history>delete beta version.......its all done....thanks me later

**Comment by Soulzityr** [comment]
There was a windows update a while ago that fixed this issue, I believe.

**Comment by lone-assassin-aj** [comment]
> There was a windows update a while ago that fixed this issue, I believe.

Yes, one of the latest Stable build updates did fix the issue at my end too. Not exactly sure which one though.
